file_formats/stimulus_tables.md

```diff
@@ -0,0 +1,118 @@
+...
+```
+
+### Application notes
+
+TODO
+This section is reserved to provide additional information on how to acquire the data in the data format described above. It can include information relative to the hardware (e.g. supported models), software interface (e.g. SoftwareFoo with version >= 1.0.0), ideally, some easy to deploy or follow examples that can get anyone to reproduce the data format.
+
+
+
+### Relationship to aind-data-schema
+
+There is no relationship between the raw data and the aind-data-schema that is relevant here. The raw stimulus information relates to the aind-data-schema through the derived asset, which is described below.
+
+
+### File Quality Assurances
+
+TODO
+This section is reserved to describe what features of the data format should be true if the data asset is to be considered valid. Conceptually, this section should describe features that can be easily tested and validated by unit tests. Examples include:
+- "There will always be two files: `data.dat` and `metadata.txt`"
+- "For each frame in `video.avi`, there will be a corresponding row in `metadata.csv`"
+- "Field `Bar` will always be a positive integer"
+- "The first timestamp value in `metadata_camera.csv` will always be greater than the first one in `metadata_behavior.csv`
+- The `Time` column in `file.bin` is assumed to be aligned (sharing the same time domain) with `Time` column of `another_file.csv`
+
+
+## Primary Data Format
+
+### File format
+
+#### Basic Stimulus Table
+The stim tables are stored as `.CSV` tables. At minimum, it must include the following columns:
+- `start_time`: float representing the time, in seconds, the stimulus began
+- `stop_time`: float representing the time, in seconds, the stimulus ended
+- `stim_name`: string representing the unique kind of stim shown. E.g "receptive_field_mapping", "drifting_gratings", "natural_movie_1"
+
+Any stimulus which is not procedurally generated should have additional columns. 
+For image sequences:
+- `image_name`: string representing the name of the image. This is used to associate the stimulus with a stim_template object/file which should have the same name in subsequent data packaging.
+- `image_index`: integer representing the position of the image in the sequence. If there is no canonical or expected order for the sequence of images this is not necessary, but may still be helpful.
+
+For movies:
+- `movie_name`: string representing the name of the movie. This is used to associate the stimulus with a stim_template object/file which should have the same name in subsequent data packaging.
+- `frame_index`: integer representing the index of the frame in the regular forward-playing movie. If the movie is scrambled or played backward, the `frame_index` should represent the position of the frame if it were in the unaltered movie.
+
+Additionally, it can be helpful to include:
+- `n_repeats`: integer representing the number of times an image sequence or movie has been repeated.
```

Comment:
a) number_repeats? 
b) this is very unique to a specific stimulus. Not sure I'd include it in a general standard.